### DISCUSSION TITLE
Save snap button

### DISCUSSION THREAD

**Discussion by Radzi11** [rodit/SnapMod#156] [Ideas & Feature Suggestions]
Nothing else needs to be added here. Is there any chance to add a snap download button on the screen while opening the snap? Something like there was in snaptool.

**Comment by Magna-Graecia** [comment]
Just use the in-snap context menu. It's literally 1 more tap compared to having an overlay button, and the best part is that nothing is hovering over any part of the image/video and getting in the way while viewing. I personally like it the way it is, but it's all up to rodit if he wants to add the button or not. I just hope that if it's added, it's added as an option that can be enabled/disabled within SnapMod settings.

**Reply by Radzi11** [reply]
Yes, if a button is added, I would also like it to be able to be turned on and off. so that everyone is satisfied

**Comment by Radzi11** [comment]
my second question is where are the photos / videos after downloading via the save button in the context menu? I have a message that support has started, but I cannot locate the photos

**Reply by Magna-Graecia** [reply]
If using the in-snap context menu (3 dots in upper right-hand corner), they get saved to Internal Storage/Movies/SnapMod by default. If using the in-chat menu where you long-press on a snap to get the context menu and then press 'Save to Camera Roll', those get saved to Internal Storage/Snapchat.

**Reply by Magna-Graecia** [reply]
Also, if you just downloaded the app, make sure to go into device settings and check that storage permissions have been granted for SnapMod. It didn't get granted by default for me, and it didn't ask for permission. At first I thought something was wrong until I went and saw that permission hadn't been granted. After granting manually, everything worked fine.

**Reply by Radzi11** [reply]
strange, despite allowing access to files, it does not save my photos in the default place ...

**Reply by RevealedSoulEven** [reply]
Totally useless

**Comment by rodit** [comment]
This is unfortunately a lot more effort than it's worth.

**Comment by jammyjay** [comment]
This was asked before and honestly I don't see a need for a button, further cluttering the screen on a snap. The methods work fine as is now Rodit.

**Comment by Radzi11** [comment]
no problem. Waiting for some official application icon: D

**Comment by Magna-Graecia** [comment]
@rodit  @jammyjay Finally, the words of reason.
